src/com/owncloud/android/ui/fragment/OCFileListFragment.java

```diff
@@ -335,60 +339,76 @@ private void removeFabLabels() {
     }
 
     private void registerLongClickListener() {
-        getListView().setOnItemLongClickListener(new AdapterView.OnItemLongClickListener() {
-            public boolean onItemLongClick(AdapterView<?> arg0, View v,
-                                           int index, long arg3) {
-                showFileAction(index);
-                return true;
+        getListView().setMultiChoiceModeListener(new AbsListView.MultiChoiceModeListener() {
```

Comment:
This is the line I'm talking about in https://github.com/owncloud/android/issues/1605#issuecomment-220543290